### DISCUSSION TITLE
Is it possible to by default aways delay when you hit 'Send'?

### DISCUSSION THREAD

**Discussion by GerrySeidman** [Extended-Thunder/send-later#295] [General]
I have been bitten too many times with noticing/thinking about something just as I hit "Send'.  It would be great when you hit send that by default messages don't actually get send for some configurable time (say 5 minutes).   My understanding is that this plugin only allows doing this if you hit 'Send Later'

Note: I haven't installed the plugin yet.  It asks for full permission for full access to the computer, so I am reluctant to do so until I know if it will actually do what I want

Thanks,
Gerry

**Comment by wuhandong** [comment]
Choose one of the two options in the attached screenshot
![image](https://user-images.githubusercontent.com/36207248/109342071-f048d380-7830-11eb-9131-9c4dabc07803.png)

**Reply by GerrySeidman** [reply]
Super.  Thanks.  

Have a great weekend
